test/jdk/java/foreign/TestLayoutConstants.java

```diff
@@ -106,6 +106,9 @@ public void testDescribeResolveFunction(MemoryLayout layout, boolean isVoid) {
                         MemoryLayout.ofStruct(
                                 MemoryLayouts.PAD_8,
                                 MemoryLayouts.BITS_32_BE)) },
+                { MemoryLayouts.BITS_32_LE.withName("myInt") },
+                { MemoryLayouts.BITS_32_LE.withBitAlignment(8) },
+                { MemoryLayouts.BITS_32_LE.withAttribute("xyz", "xyz") },
```

Comment:
Maybe change the test to have name != value, so that we also detect bugs where we serialize the name for both.

Reply:
Good idea!